Mathlib/FieldTheory/IsAlgClosed/AlgebraicClosure.lean

```diff
@@ -416,4 +416,25 @@ instance [CharZero k] : CharZero (AlgebraicClosure k) :=
 instance {p : ℕ} [CharP k p] : CharP (AlgebraicClosure k) p :=
   charP_of_injective_algebraMap (RingHom.injective (algebraMap k (AlgebraicClosure k))) p
 
+/-09GY Lemma: Two polynomials in $k[x]$ are relatively prime
+precisely when they have no common roots in an algebraic closure $\overline{k}$ of $k$.
+[Stacks: Lemma 09GY](https://stacks.math.columbia.edu/tag/09GY) -/
+lemma IsComrime_iff_no_common_root {k : Type*} [Field k] {p q : Polynomial k} :
```

Comment:
The name should change to `isComrime_iff_no_common_root`. You can read [naming conventions](https://leanprover-community.github.io/contribute/naming.html#general-conventions) to know how to name theorems correctly.